stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -471,6 +471,33 @@ func.func @scatter(%input_tensor: tensor<200x100x300xf32>,
 
 // -----
 
+// CHECK-LABEL: func @scatter_bounds
+func.func @scatter_bounds(%input_tensor: tensor<200x?x?xf32, #stablehlo.type_extensions<bounds = [?, 101, 301]>>,
```

Comment:
nit: the last 2 dims are testing the same thing, how about we change to:
tensor<200x?x?xf32, #stablehlo.type_extensions<bounds = [?, ?, 301]>>

Reply:
Good point!